Remotion/Data/DomainObjects.UnitTests/Mapping/MappingReflectionIntegrationTests/MappingReflectionIntegrationTestBase.cs

```diff
@@ -48,6 +48,13 @@ public virtual void SetUp ()
       _typeDefinitions = typeDefinitions;
     }
 
+    [OneTimeSetUp]
+    public virtual void OneTimeSetUp ()
+    {
+      TestMappingConfiguration.Initialize();
+      StandardConfiguration.Initialize();
+    }
+
```

Comment:
It feels like we should have an Uninitialize-API, too. Otherwise, the static field is always available in later tests. Unitialize (better name?) should name Instance no longer return the value and Initialize can in turn set the same instance once more. This would bring much better isolation.

Reply:
While that is true, I don't think that we can do any better. Creating a `MappingConfiguration` costs time and re-creating it every time a fixture ends will increase the test runtime by a lot. The main point here is just that if you run one of the `When_XXX` tests in isolation the test would fail because the `StandardConfiguration` is not set up.

We have the same problem of uninitialization in the `DatabaseTest`. I think that this is not a problem as the configuration is always the same for all the tests and we stop the important mistake of accessing the database without inheriting from `DatabaseTest`.

Reply:
Miscummunication: I don't want to re-create it. Just block access to the field during the unitialized-phase / remove it from MappingConfig.Current, etc. That way, we don't leak. It wasn't possible before and if we can do it now cheaply, it would be good to do it right now. 

Otherwise, we may want to call it `EnsureInitialized`

Reply:
We can do this but I would prefer to do it in another issue.